Pod/Classes/WPMediaCollectionViewController.m

```diff
@@ -657,13 +660,15 @@ - (void)showAlertAboutMediaCapturePermission
 
 - (void)processMediaCaptured:(NSDictionary *)info
 {
+    NSInteger mediaItemsBefore = [self.dataSource numberOfAssets];
     self.ignoreMediaTimestamp = [NSDate timeIntervalSinceReferenceDate];
     WPMediaAddedBlock completionBlock = ^(id<WPMediaAsset> media, NSError *error) {
         if (error || !media) {
-            NSLog(@"%@", error);
+            NSLog(@"Adding media failed: %@", [error localizedDescription]);
```

Comment:
We shouldn't have any NSLog statements - use DDLogError so they show up in the contain app's logs.